docs/publishing/sections/4_notebooks_styling.md

```diff
@@ -185,39 +189,42 @@ Please note:
 
     ![header format](../assets/section4_image1.png)
 
-#### Code Cell:
+#### Code Cell
 
 You will need to create two separate code cells that take on the parameter. Let's use `district` as an example parameter once again.
 
-- Code Cell #1:
+- <b>Code Cell #1</b>
 
   - Add in your parameter and set it equal to any valid value.
 
   - Comment out the cell.
 
-  - This is how your code cell should look:
+  - This is how your code cell must look.
 
-    ```
+    ```python
     # district = "4"
     ```
 
   - Turn on the parameter tag: go to the code cell go to the upper right hand corner -> click on the gears -> go to "Cell Tags" -> Add Tag + -> add a tag called "parameters" -> click on the new "parameters" tag to ensure a checkmark shows up and it turns dark gray.
 
     ![parameters tag](../assets/section4_image2.png)
 
-- Code Cell #2:
+- <b>Code Cell #2</b>
 
   - Input the same parameter without any assigned value with `%%capture_parameters` at the top.
-  - This is how your code cell should look:
-    ```
-    %%capture_parameters
-    district
-    ```
+  - Your code cell  must <b>exactly</b> like this or else your notebook won't parameterize.
```

Comment:
afaik it is technically possible to have more code/comments in one of these cells, just as long as `%%capture_parameters` remains at the very top of the cell. For example speedmaps has 2 of these, with an extra line of code: https://github.com/cal-itp/data-analyses/blob/main/ca_transit_speed_maps/speedmaps.ipynb

Perhaps we should rephrase this to a recommendation?

Reply:
Yes I'll rephrase this. Thanks for checking this out